EIPS/eip-wrapped_token.md

```diff
@@ -0,0 +1,124 @@
+---
+eip: <to be assigned>
+title: Wrapped token inspection
+author: David Mihal (@dmihal)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category (*only required for Standard Track): ERC
+created: 2020-08-05
+---
+
+## Simple Summary
+Defines a standard interface for viewing the addresses and balances of wrapped tokens.
+
+## Abstract
+Many projects have emerged that create new assets by wrapping existing tokens. However, it can be difficult for applications to easily query which tokens are wrapped, and how much of each token is associated with the holder of the wrapper token.
+
+This EIP introduces a simple interface for exposing the addresses of all wrapped tokens, as well as querying wrapped token balances.
+
+## Motivation
+Many projects, particularly in decentralized finance, have created new assets by wrapping existing tokens. This includes, but is not limited to:
+
+* Liquidity pool tokens for automated market makers (Uniswap, Balancer)
+* Interest-bearing lending tokens (Compound cTokens, Aave aTokens)
+* Managed investment tokens (Set Protocol, yEarn vaults)
+* Utility wrapper tokens (DeFi777)
```

Comment:
```suggestion

* Liquidity pool tokens for automated market makers
* Interest-bearing lending tokens
* Managed investment tokens
* Utility wrapper tokens
```
Names of specific projects in an EIP isn't likely to age well, so it is best to leave them out.  It may be worth giving a *very short* description of how each of these things wraps tokens and why as well.  For example, I don't know what a "utility wrapper token" is or why it needs wrapping or how it wraps.